scripts/exglobal_stage_ic.sh

```diff
@@ -8,6 +8,28 @@ GDATE=$(date -d "${PDY} ${cyc} - ${assim_freq} hours" +%Y%m%d%H)
 gPDY="${GDATE:0:8}"
 gcyc="${GDATE:8:2}"
 
+# Determine NMEM_ENS based on the configuration file (e.g., config.base)
+# Assuming NMEM_ENS is defined in config.base as an integer
+
+config_file="${EXPDIR}/config.base"
+
+if [[ -e "${config_file}" ]]; then
+    NMEM_ENS=$(grep "NMEM_ENS" "${config_file}" | awk -F= '{print $2}' || true)
+else
+    echo "Error: ${config_file} does not exist or is not accessible." >&2
+    exit 1
+fi
```

Comment:
What is all this for? `config.base` is already sourced by every job, so NMEM_ENS will be set.

Reply:
ok. will remove it